UefiPayloadPkg/Tools/MkFitImage.py

```diff
@@ -54,6 +54,7 @@ def BuildConfNode(Fdt, ParentNode, MultiImage):
 
     Fdt.setprop(ConfNode1, 'require-fit', b'', 0)
     Fdt.setprop_str(ConfNode1, 'firmware', 'tianocore')
+    Fdt.setprop_str(ConfNode1, 'loadables', 'uefi-fv\0bds-fv')

```

Comment:
This is a good idea, but we should do it properly by iterating `MultiImage` in `BuildFitImage()` below